lib/util/DiagramWalkerUtil.js

```diff
@@ -101,6 +101,10 @@ export function participatesIn(participant, shape) {
  */
 export function getAllElementsOfType(type, choreos) {
   function traverse(node) {
+    if (node.flowElements) {
```

Comment:
Haven't tested it, but isn't the condition the wrong way round?

Reply:
True